### PR TITLE
Fix: mini profile widget

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- show mini profile widget
- fix layout widget translations extraction
- update widget area toggling

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Comment by SeverS** [comment]
Top alignment of the topbar and widget area is broken
Screenshot 2023-12-11 at 19 52 26

**Comment by SeverS** [comment]
the mini profile widget doesn't load on the preview

**Comment by josenriagu** [comment]
> the mini profile widget doesn't load on the preview

It seems to be a bit buggy, at times it doesn't load until you refresh. So I will take a look at the activeWhen method in the register function

File: ui/widgets/mini-profile/src/index.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ export const register: (opts: IntegrationRegistrationOptions) => WidgetInterface
   mountsIn: opts.layoutConfig?.widgetSlotId,
   loadingFn: () => import('./mini-profile-widget'),
   activeWhen: (location, pathToActiveWhen) => {
-    return pathToActiveWhen('/@akashaorg/app-akasha-integration/post/', false)(location);
+    return (
+      pathToActiveWhen('/@akashaorg/app-akasha-integration/beam/', false)(location) ||
```
Comment (kenshyx) [inline ui/widgets/mini-profile/src/index.ts]: Added via Giphy

File: ui/design-system-components/src/components/Loaders/mini-profile-widget-loader.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const MiniProfileWidgetLoader: React.FC = () => {
+  return (
+    <Card radius="rounded-2xl" margin="mb-4" padding="p-0" customStyle="max-h-[30rem]">
+      <Stack align="center" customStyle="h-28 rounded-t-2xl" fullWidth>
+        <Stack customStyle="relative top-12">
+          <CircularPlaceholder height="h-20" width="w-20" customStyle="shrink-0" animated />
+        </Stack>
+      </Stack>
+
+      <Stack spacing="gap-y-6" align="center" padding="p-4 pt-6" fullWidth>
```
Comment (didd) [inline ui/design-system-components/src/components/Loaders/mini-profile-widget-loader.tsx]:
```suggestion
      <Stack spacing="gap-y-4" align="center" padding="p-4" customStyle="pt-6" fullWidth>
```

File: ui/design-system-components/src/components/ProfileMiniCard/index.tsx [hunk 3]
```diff
@@ -1,144 +1,110 @@
...
-              {/* {(profileData.userName && `@${profileData.userName}`) || */}
-              {truncateMiddle(profileData.did.id)}
+            profileId={profileData?.did?.id}
+            customStyle="cursor-pointer"
+            onClick={handleClick}
+          />
+        </Stack>
+      </Stack>
+
+      <Stack spacing="gap-y-6" align="center" padding="p-4 pt-6">
```
Comment (didd) [inline ui/design-system-components/src/components/ProfileMiniCard/index.tsx]:
```suggestion
       <Stack spacing="gap-y-4" align="center" padding="p-4" customStyle="pt-6">
```

**Comment by didd** [comment]
The routing issue requires a fix. The mini profile doesn't load until it's refreshed. With this note approving this PR.

**Review by didd — APPROVED** [review]

**Pull request merged**